### DISCUSSION TITLE
Help with placements/shape overrides?

### DISCUSSION THREAD

**Discussion by aboweFIS** [OrchardCMS/OrchardCore#16719] [Q&A] [answered]
**I made a content type called "SingleTitledText" for fields I found myself recreating over and over:**

![Screenshot 2024-09-11 103317](https://github.com/user-attachments/assets/c1ae27fb-2a13-499e-859f-a5e78c1b6081)

**I'm using it in BagParts in other content types, but I don't always need ALL the fields. For instance, in the following, I only need "Heading" and "Detail Text" to display as options, and I want to hide the others:**

![Screenshot 2024-09-11 103603](https://github.com/user-attachments/assets/ad15bc7c-c4cd-4c4b-b72b-584ee3881b79)
![Screenshot 2024-09-11 103629](https://github.com/user-attachments/assets/9a1b65e9-9656-494e-b420-ffbb1901fd75)

**I think I can use placements to show/hide the fields I need but I can't get the file name correct (or am otherwise doing something wrong).**

**Also on these I'd like to remove the option to add more than one which I think I can do in the BagPart.cshtml View, but again, I can't figure out the correct template name to give it.**
![image](https://github.com/user-attachments/assets/f98e314c-157f-486e-9be8-817f5a926b3f)

**I've studied [Templates](https://docs.orchardcore.net/en/latest/reference/modules/Templates/) and have watched the different Dojo videos but it's still clear as mud. Note I'm coming to this as a front-end designer/developer so it's been a bit of a challenge! Thanks in advance!**

**Comment by Piedone** [comment]
Since it seems to me you're just bundling a few fields together in the type, the easiest would be to just use different types for the different use cases. This works if the distinction for "I don't always need ALL the fields" is different parent content types. If not, what is it?

**Reply by aboweFIS** [reply, marked as answer]
Okay so you're saying I'm going about it the wrong way. I thought this is exactly why I'd use alternates and placements but it sounds like I am just making it harder on myself. I'll go back to the drawing board. Thanks!
